pkg/vm/ch.go

```diff
@@ -17,9 +19,43 @@ import (
 )
 
 const (
-	chBin = "cloud-hypervisor"
+	chBin           = "cloud-hypervisor"
+	cloudConsoleBin = "cloud-console"
 )
 
+// StartCloudConsole Starts the cloud console for the vm on it's private network ip
+func (m *Machine) StartCloudConsole(ctx context.Context, namespace string, networkAddr net.IPNet, machineIP net.IPNet, ptyPath string) error {
```

Comment:
Why this function is defined on a Machine receiver not on the Manager. It does not access any of the Machine properties.

I think it's better for this to become a `private` manager method instead it fits more there.